flink-table/flink-table-api-scala/src/main/scala/org/apache/flink/table/api/ImplicitExpressionConversions.scala

```diff
@@ -496,16 +496,16 @@ trait ImplicitExpressionConversions {
   }
 
   /**
-   * Converts the date time string with format 'yyyy-MM-dd HH:mm:ss' under the 'UTC+0' time zone
-   * to a timestamp value of [[DataTypes.TIMESTAMP]].
+   * Converts the date time string with format 'yyyy-MM-dd HH:mm:ss' under the 'UTC+0' time zone to
+   * a timestamp value of [[DataTypes.TIMESTAMP]].
```

Comment:
What is this change about?

Reply:
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=41747&view=logs&j=52b61abe-a3cc-5bde-cc35-1bbe89bb7df5&t=54421a62-0c80-5aad-3319-094ff69180bb 

The code style check failed.

Reply:
CI is passing though?

Reply:
I also don't get it locally...

Reply:
This was already fixed in 82e279ee.